src/cbmc_starter_kit/template-for-repository/proofs/run-cbmc-proofs.py

```diff
@@ -216,6 +228,12 @@ def run_build(litani, jobs, fail_on_proof_failure):
         logging.critical("Failed to run litani run-build")
         sys.exit(1)
 
+    if tabular_summary_print_fmt:
+        with open(out_file_path) as fp:
+            run_dict = json.load(fp)
```

Comment:
Can I suggest more robust error handling here? If for any reason the summary could not be printed out, I think that shouldn't be a fatal error, just print out a warning/error and exit.